### DISCUSSION TITLE
AdFly is dead what now?

### DISCUSSION THREAD

**Discussion by BhaskarPanja93** [BhaskarPanja93/View-Bot-Client#17] [Announcements]
Dont worry folks, we used to earn while AdFly was a thing, we will even when it isn't. Basically within a month, i will start working on some AdFly alternative that works best for us, and soon everything will be back again. Please have patience. My exams end December 20th, will start working on the new bot then. Thank you for your cooperation

**Comment by Dec1lent** [comment]
My last exam ends on december 20th too, I still have the Linkvertise bot, but sometimes I get banned and other times I wont, its not about speed at this point. But the last thing to change would be having tons of proxies, and random clicks on the site.

**Comment by BhaskarPanja93** [comment]
@Dec1lent alright i am done, ready to finish the bot asap, lets distribute the work a bit, u make me a selenium instance, that has all the settings, like push ads enabled, and such. i'll make the click process... also i'll do the " make a new VM " thing with every instance..

**Reply by Luskafe** [reply]
Yay finally ! you're back looking forward on the new bot !

**Comment by Andres-Fernandez-Caballero** [comment]
hi every one ✋  this project interes me a lot if i can help you in something tell me 
sorry my english i am from argentina 😄
